libs/hmpb/src/ballot_templates/nh_ballot_template.tsx

```diff
@@ -102,6 +102,13 @@ function Header({
       ? assertDefined(getPartyForBallotStyle({ election, ballotStyleId }))
       : undefined;
 
+  const clerkSignatureImage =
+    clerkSignatureImageOverride ?? election.signatureImage;
+  // clerkSignatureCaptionOverride will be an empty string if it is
+  // entered and then deleted, so treat it as undefined
+  const clerkSignatureCaption =
+    clerkSignatureCaptionOverride || election.signatureCaption;
```

Comment:
The `election.signatureImage` and `election.signatureCaption` are still treated as optional in this template for backwards compatibility. In the future we could `assert` these fields